di_website/publications/static/publications/css/style.css

```diff
@@ -0,0 +1,31 @@
+.accordion-leaf {
+  background-color: #109E67;
+}
+
+.accordion-poppy{
+  background-color: #e84439;
+}
+
+.accordion-bluebell {
+  background-color: #0089CB;
+}
+
+.accordion-rose {
+  background-color: #C2145B;
+}
+
+.accordion-sunflower {
+  background-color: #F49B20;
+}
+
+.accordion-marigold {
+  background-color: #EB632B;
+}
+
+.accordion-lavendar {
+  background-color: #89328F;
+}
+
+.accordion-spacing .accordion__list-item:first-child {
+  margin-top: 0;
+}
```

Comment:
You'll have to change these to be dependent on the body's thematic classes i.e.
`.body--rose .accordion__heading {...}`